src/app/add-course-page/add-course-page.component.html

```diff
@@ -0,0 +1,97 @@
+<div class="login-wrapper">  
+    <mat-card class="login-card">
+      <mat-card-header>
+        <mat-card-title>New course</mat-card-title>
+      </mat-card-header>
+      <mat-card-content>
+        <form  class="add-course-form">
+            <table class="add-course-form-full-width" cellspacing="0">
```

Comment:
please do not use the table for layout, it's 2020, choose different approach

Reply:
okay, I changed this one